__tests__/app.test.js

```diff
@@ -34,3 +34,34 @@ describe("GET /api/categories", () => {
       });
   });
 });
+describe("GET /api/reviews", () => {
+  test("returns an array of review objects that includes comment_count properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(Object.keys(body)).toEqual(["reviews"]);
+
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
```

Comment:
We don't need to check to see if it's an instance of an array as we do this implicitly by checking the length and then looping through the array